plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchMetadata.java

```diff
@@ -110,16 +105,13 @@
 import static io.trino.spi.type.VarbinaryType.VARBINARY;
 import static io.trino.spi.type.VarcharType.VARCHAR;
 import static java.lang.String.format;
-import static java.nio.charset.StandardCharsets.UTF_8;
 import static java.util.Collections.emptyIterator;
 import static java.util.Locale.ENGLISH;
 import static java.util.Objects.requireNonNull;
 
 public class ElasticsearchMetadata
         implements ConnectorMetadata
 {
-    private static final ObjectMapper JSON_PARSER = new ObjectMapperProvider().get();
-
     private static final String PASSTHROUGH_QUERY_SUFFIX = "$query";
```

Comment:
I would remove this all together. I guess the error message that table is not found is good enough.